src/query/config/src/inner.rs

```diff
@@ -71,6 +71,14 @@ impl Config {
         Ok(cfg)
     }
 
+    /// # NOTE
+    ///
+    /// This function is served for tests only.
+    pub fn load_without_args() -> Result<Self> {
```

Comment:
If this is only used in tests, can we add an `#[cfg(test)]` for it?

Reply:
```rust
error[E0599]: no function or associated item named `load_for_test` found for struct `databend_meta::configs::Config` in the current scope
  --> src/meta/service/tests/it/configs.rs:71:27
   |
71 |         let cfg = Config::load_for_test().expect("load must success");
   |                           ^^^^^^^^^^^^^ function or associated item not found in `databend_meta::configs::Config`
```

Not works for tests that not in the same crate.

Reply:
> Not works for tests that not in the same crate.

At least, we learn more about rust.